src/Polysemy/Scoped.hs

```diff
@@ -26,75 +91,34 @@ import Polysemy.Internal.Tactics (Tactical, runTactics)
 -- @withResource@ is a callback function, allowing the user to acquire the resource for each program from other effects.
 --
 -- @scopedInterpreter@ is a regular interpreter that is called with the @resource@ argument produced by @scope@.
+-- This is mostly useful if you want to reuse an interpreter that you cannot easily rewrite (like from another library).
+-- If you have full control over the implementation, 'interpreteScoped' should be preferred.
 --
 -- /Note/: This function will be called for each action in the program, so if the interpreter allocates any resources,
 -- they will be scoped to a single action. Move them to @withResource@ instead.
```

Comment:
Should this warning be moved to the type perhaps? Or added to all the other combinators that do this as well?

Reply:
no, that is specific to `runScoped` because it uses a blackbox interpreter.
with a handler, the problem cannot occur.